### ISSUE TITLE
Driver: DuckDB 🦆

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#33384]
**Is your feature request related to a problem? Please describe.**
Opening the official feature request for a DuckDB driver

**Describe the solution you'd like**
An officially supported driver for DuckDB

**Describe alternatives you've considered**
NA

**How important is this feature to you?**
NA

**Additional context**
NA

**Comment by tovbinm** [comment]
Great idea!!

**Comment by paoliniluis** [comment]
cuack cuack
![image](https://github.com/metabase/metabase/assets/1711649/1478a5cb-cd0b-4e5b-bf54-f9a7999c9152)

https://github.com/metabase/metabase/issues/43338

**Comment by sugendran** [comment]
🤔 @paoliniluis does this mean duckdb is landing as a connection choice in the cloud version?

**Comment by dragonsahead** [comment]
@sugendran you can use it on sef hosted instances, for now

**Comment by jonshubeli** [comment]
Yes, a connector to DuckDB and MotherDuck would be highly valuable.

**Comment by tovbinm** [comment]
We have been using this one for a while and it works well - https://github.com/faros-ai/duckdb-java
